Gemfile

```diff
@@ -44,18 +47,20 @@ group :development, :test do
 
   # rspec
   gem 'rspec-rails', '~> 3.5'
-  gem 'shoulda-matchers'
 
   # RSpec::JsonExpectations
   gem 'rspec-json_expectations'
 end
 
 group :test do
-  # for build strategies
-  gem 'factory_girl_rails'
-
-  # database cleaner
-  gem 'database_rewinder'
+  # a library for setting up Ruby objects as test data.
+  gem 'factory_girl_rails', '~> 4.0'
```

Comment:
I specified the lowest version in `Gemfile.lock`. I am worried about inhibiting version up by writing this.